spec/datadog/tracing/span_operation_spec.rb

```diff
@@ -83,7 +83,8 @@
           start_time: nil,
           status: 0,
           trace_id: kind_of(Integer),
-          type: nil
```

Comment:
You'll want to add a test to the `given an option` group for `links`: https://github.com/DataDog/dd-trace-rb/blob/7e9fe1efbdc6c14b9eb14fe004684329cc45baaf/spec/datadog/tracing/span_operation_spec.rb#L103-L289

Reply:
I added test coverage for the links field in span_operation